src/app/service/request.tsx

```diff
@@ -0,0 +1,10 @@
+export default function useRequests(){
```

Comment:
naming with "use" prefix is for hooks not constants

Reply:
ok  i will remove "use" prefix .